presto-main/src/main/java/com/facebook/presto/connector/system/NodeSystemTable.java

```diff
@@ -44,7 +50,7 @@
             .column("http_uri", VARCHAR)
             .column("node_version", VARCHAR)
             .column("coordinator", BOOLEAN)
-            .column("active", BOOLEAN)
+            .column("state", VARCHAR)
```

Comment:
This is mostly informational (for users), so I don't think it's a big issue to change it.